src/pybind/mgr/dashboard/frontend/src/app/shared/api/configuration.service.ts

```diff
@@ -10,6 +10,27 @@ import { ApiModule } from './api.module';
 export class ConfigurationService {
   constructor(private http: HttpClient) {}
 
+  private findValue(config, section: string) {
+    return config.value.find((v) => v.section === section);
```

Comment:
On a clean cluster `value` is `undefined` and this line throws an exception.
[pg.log](https://github.com/ceph/ceph/files/3810268/pg.log)

Reply:
Hit the same issue as @tspmelo mentions, and got the spinner forever:

![captured (8)](https://user-images.githubusercontent.com/37327689/68230658-c1826a80-fff9-11e9-9808-617d643a5267.gif)

Reply:
Same here in my local vstart test environment. Clicking on "Create" never shows the pool creation dialogue.

Reply:
It should be fixed now. Thanks.